src/main/scala/org/apache/mesos/chronos/notification/SlackClient.scala

```diff
@@ -21,7 +21,7 @@ class SlackClient(val webhookUrl: String) extends NotificationClient {
     generator.writeStartObject()
 
     if (message.nonEmpty && message.get.nonEmpty) {
```

Comment:
I was following MailClient and RavenClient, both of which seem to assume that the subject is present.  I can add another check, however, if you would like.